packages/blocks/src/utils/get-styles/getLayoutStyles.ts

```diff
@@ -0,0 +1,53 @@
+import { BlocksTheme } from '../../types/theme.js';
+import { BlockWithAttributes } from '../../components/WordPressBlocksViewer.js';
+
+/**
+ *
+ * Returns the specified block Layout Styles
+ * @param theme Block Theme object
+ * @returns React CSS Properties object
+ */
+export default function getLayoutStyles<T extends BlockWithAttributes>(
```

Comment:
A nice to have feature to handle custom layout columns:

<img width="1436" alt="Screenshot 2023-06-01 at 16 58 26" src="https://github.com/wpengine/faustjs/assets/328805/20a581ad-03ec-4ce7-abc1-f8f949622eb3">